src/toolbar_gui.cpp

```diff
@@ -1047,7 +1061,7 @@ static CallBackFunction MenuClickForest(int index)
 
 static CallBackFunction ToolbarMusicClick(Window *w)
 {
-	PopupMainToolbMenu(w, _game_mode == GM_EDITOR ? (int)WID_TE_MUSIC_SOUND : (int)WID_TN_MUSIC_SOUND, STR_TOOLBAR_SOUND_MUSIC, 1);
+	PopupMainToolbMenu(w, _game_mode == GM_EDITOR ? (int)WID_TE_MUSIC_SOUND : (int)WID_TN_MUSIC_SOUND, {STR_TOOLBAR_SOUND_MUSIC});
```

Comment:
Why the `(int)` casting when a ternary is used, here and in `ToolbarHelpClick()`? Other uses don't seem to need it.

Reply:
Two different enums can't be used in a ternary. The others do variably cast to byte, uint or int which is inconsistent, but I'm not touching this in this PR.